hbase-client/src/main/java/org/apache/hadoop/hbase/quotas/RpcThrottlingException.java

```diff
@@ -130,10 +130,19 @@ public static void throwWriteCapacityUnitExceeded(final long waitInterval)
 
   private static void throwThrottlingException(final Type type, final long waitInterval)
     throws RpcThrottlingException {
-    String msg = MSG_TYPE[type.ordinal()] + MSG_WAIT + StringUtils.formatTime(waitInterval);
+    String msg = MSG_TYPE[type.ordinal()] + MSG_WAIT + stringFromMillis(waitInterval);
     throw new RpcThrottlingException(type, waitInterval, msg);
   }
 
+  private static String stringFromMillis(long millis) {
+    if (millis >= 1000) {
+      return StringUtils.formatTime(millis);
```

Comment:
Actually, I realize this will continue to be misleading for values greater than 1s. For example 1.9s is almost double what would be shown by formatTime. It matters less for values in the 10s of seconds or higher, but I think it will be very common to have values mostly in the single digit seconds. 

We could copy the impl of formatTime here and add ms, or use the existing formatTime and append a ms part, or look for a different "humanize" method/library (as long as it's already a dep)